automated install/basic-install.sh

```diff
@@ -88,11 +88,14 @@ if [[ $(command -v apt-get) ]]; then
   # fixes for dependancy differences
   # Debian 7 doesn't have iproute2 use iproute
   ${PKG_MANAGER} install --dry-run iproute2 > /dev/null 2>&1 && IPROUTE_PKG="iproute2" || IPROUTE_PKG="iproute"
-  # Prefer the php metapackage if it's there, fall back on the php5 pacakges
-  ${PKG_MANAGER} install --dry-run php > /dev/null 2>&1 && phpVer="php" || phpVer="php5"
```

Comment:
It should be checked if my conflict resolution was correct here (removed [old line 92](https://github.com/pi-hole/pi-hole/pull/731/files#diff-689793e68cde6270d074695b71c969afL92), added [new line 93](https://github.com/pi-hole/pi-hole/pull/731/files#diff-689793e68cde6270d074695b71c969afR93))